### ISSUE TITLE
New Pod release with last commits

### ISSUE THREAD

**Issue by fegabe** [laptobbe/TSMarkdownParser#57]
Hi,

we're using this library for an appleTV app and pointing directly to the latest commit of the repo to be using the latest improvements like this: https://github.com/laptobbe/TSMarkdownParser/commit/01be140cb88d903b46d73cc883c93263cf128776
Are you planning to create a new Pod release?

Thanks

**Comment by Coeur** [comment]
Yes, I'll release version #3 this weekend. Beware, it will have breaking changes.

**Comment by fegabe** [comment]
Great thanks!

**Comment by Sianay** [comment]
Hi @fegabe any update on this ?

**Comment by fegabe** [comment]
I didn't try the last release. I was using the commit I pointed out and worked well
